packages/tao/src/shared/tsconfig.ts

```diff
@@ -0,0 +1,21 @@
+export interface CompilerOptions {
```

Comment:
I'd move this to hasher.ts and only leave the properties that are relevant (e.g., paths)